src/es6Remx/remx.js

```diff
@@ -70,8 +70,8 @@ function isDev() {
 }
 
 export function toJS(data) {
-  console.warn(`remx.toJS() is deprecated. Please remove it from your code ASAP. 
-    Be aware that things can break after removing it, most of the time because of problematic data flow. 
+  console.warn(`remx.toJS() is deprecated. Please remove it from your code ASAP.
+    Be aware that things can break after removing it, most of the time because of problematic data flow.
     Please take your time to investigate the root of cause in case of a problem, toJS() is an expansive action`);
-  return cloneDeep(data);
+  return mobx.toJS(data);
```

Comment:
is this call works when you give it something that is not mobx observable?

Reply:
`cloneDeep` comes from lodash, it doesn't know about observables, so, sure

Reply:
Sorry, I mean, toJS does deep cloning for regular data structures, so it should work the same way as cloneDeep https://github.com/mobxjs/mobx/blob/master/src/api/tojs.ts